docs/en/concepts-and-designs/lal.md

```diff
@@ -240,6 +240,80 @@ metrics:
     exp: http_response_time.sum(['le', 'service', 'instance']).increase('PT5M').histogram().histogram_percentile([50,70,90,99])
 ```
 
+### SlowSql
+
+SlowSql aims to convert LogData to DatabaseSlowStatement. It extracts data from `parsed` result and save them as DatabaseSlowStatement. SlowSql will not abort or edit logs, you can use other LAL for further processing.
+We need to set a log tag("isSlowSql") to enable it. If logs sent to OAP does not have this tag, slowSql processing will be skipped.The example to set the tag is following:
+``` json
+[
+   {
+      "tags":{
+         "data":[
+            {
+               "key":"isSlowSql",
+               "value":"true"
```

Comment:
I am thinking about the naming of this tag. As an official tag, we should define it as

`key`=`LOG_KIND` `value`=`SLOW_SQL`.

Could you adjust codes and docs to this?

Reply:
I get it.